examples/StorageClientExample.py

```diff
@@ -23,6 +24,7 @@
 
 def scan_edge(space, return_cols, all_cols):
     scan_edge_response_iter = storage_client.scan_edge(space, return_cols, all_cols, 100, 0, sys.maxsize)
+    print('############## scanned edge data ##############')
```

Comment:
Do you mind using `LOG` to replace `print` ?

Reply:
Good idea!